submissions/samuelJegede/database/querying transport database.sql

```diff
@@ -0,0 +1,32 @@
+-- Number of passengers that survived
+SELECT COUNT(survived) FROM accidents WHERE survived = 1;
```

Comment:
This is wrong. Confirm the data you imported are correctly placed in the right columns. Also COUNT(survived) can be COUNT(*)